### PR TITLE
sdk/node: add separate transaction feed test

### PR DESCRIPTION
**Pull request by jeffomatic**

### CODE REVIEW

File: sdk/node/test/transactionFeeds.js [hunk 1]
```diff
@@ -0,0 +1,144 @@
+/* eslint-env mocha */
+
+const chain = require('../dist/index.js')
+const uuid = require('uuid')
+const assert = require('assert')
```
Comment (chrisgarvin) [inline sdk/node/test/transactionFeeds.js]: you can rid of lines 3-5, since they are unused

File: sdk/node/test/transactionFeeds.js [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+  createAsset,
+  signer
+} = require('../testHelpers/util')
+
+const buildSignSubmit = buildFunc =>
+  client.transactions.build(buildFunc)
+    .then(tpl => signer.sign(tpl))
+    .then(tpl => client.transactions.submit(tpl))
+
+describe('Transaction feeds', () => {
```
Comment (chrisgarvin) [inline sdk/node/test/transactionFeeds.js]: Maybe switch to singular `Transaction feed` to stay consistent with the other test files

File: sdk/node/test/transactionFeeds.js (hunk 1 above)
Comment (jeffomatic) [inline sdk/node/test/transactionFeeds.js]: Nice catch

File: sdk/node/test/transactionFeeds.js (hunk 2 above)
Comment (jeffomatic) [inline sdk/node/test/transactionFeeds.js]: Will do, although I notice that the file name itself is plural, so we should pick one noun style for everything and stick with it.

**Pull request merged**